plugins/Hosting/js/host.js

```diff
@@ -46,7 +46,15 @@ function buildProps(hostInfo) {
 function update(hostInfo) {
 	// Store hostInfo results property data
 	$.each(hostInfo, function(name, value) {
-		props[name].value = value;
+		try {
+			props[name].value = value;
+		} catch (e) {
+			// 'name' from hostInfo probably didn't match a member of the props
+			// object
+			console.error(e);
+			console.log('Error updating host properties object:');
+			console.log(hostInfo, props);
+		}
```

Comment:
It makes it a bit more fault-tolerant. I noticed that if just one `name` from the returned JSON, `hostInfo` was off, all properties wouldn't update. This makes it so only that one property doesn't update and the host still works for the most part.